src/Neha/BankClient.java

```diff
@@ -0,0 +1,18 @@
+/* Create Bank class with fields account_no, name and amount .Create the following methods: Make it menu Driven Program using switch.
+Use Scanner class for taking inputs from User
+a) insertData()... [To save account_no, name and amount given by user].
+b) deposit()
+c) withdraw()
+d) checkBalance()
+e) displayDetails()
+Create BankClient class to test functionalities of Bank class.
+*/
+package Neha;
+
+public class BankClient {
+	public static void main(String[] args) {
+		Bank b = new Bank();
```

Comment:
Hi @NehaAug2019 
here we have to design one system like we provide option like CreditsAmount, DebitAmount
user select any method from switch case or you can give option 
like if select 
1 for --> credits Amount
2 for --> debit Amount etc..

Reply:
@Harsh9846 
This BankClient class is calling Bank class which has all the implementations of switch case